external/qt/package.cmake

```diff
@@ -24,7 +24,11 @@ find_package(Qt5 COMPONENTS Core Gui Widgets Network REQUIRED)
 find_package(Qt5Gui COMPONENTS QWebpPlugin REQUIRED)
 
 if (LINUX)
-    find_package(Qt5 COMPONENTS WaylandClient REQUIRED)
+    if (DESKTOP_APP_DISABLE_WAYLAND_INTEGRATION)
+        find_package(Qt5 OPTIONAL_COMPONENTS WaylandClient QUIET)
+    else()
```

Comment:
DESKTOP_APP_DISABLE_DBUS_INTEGRATION was made that way since if Qt is built with dbus and we won't link dbus in static build, the build will fail. Wayland is implemented completely by plugins in Qt, so this shouldn't be needed.

```
    if (NOT DESKTOP_APP_DISABLE_WAYLAND_INTEGRATION)
        find_package(Qt5 COMPONENTS WaylandClient REQUIRED)
        find_package(Qt5 OPTIONAL_COMPONENTS XkbCommonSupport QUIET)
    endif()
```

Reply:
Thanks, I wasn't aware of this.